code/modules/shuttle/spaceship_navigation_beacon.dm

```diff
@@ -0,0 +1,76 @@
+/obj/item/circuitboard/machine/spaceship_navigation_beacon
+	name = "Bluespace Navigation Gigabeacon (Machine Board)"
+	build_path = /obj/machinery/spaceship_navigation_beacon
+	req_components = list()
+
+
+/obj/machinery/spaceship_navigation_beacon
+
+	name = "Bluespace Navigation Gigabeacon"
+	desc = "A device that creates a bluespace anchor that allow ships jump near to it."
+
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "core"
+
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 0
+
+	var/id = ""
+	var/access_code = ""
+
+	density = TRUE
+	circuit = /obj/item/circuitboard/machine/spaceship_navigation_beacon
+
+
+/obj/machinery/spaceship_navigation_beacon/Initialize()
+	. = ..()
+	SSshuttle.beacons |= src
+
+/obj/machinery/spaceship_navigation_beacon/random/Initialize()
```

Comment:
Don't have a seperate type for this. Just dont assign anything to the id or access code if it has been edited